custom_components/zwave_mqtt/__init__.py

```diff
@@ -89,31 +94,52 @@ def async_node_changed(node):
     @callback
     def async_node_removed(node):
         _LOGGER.debug("[NODE REMOVED] node_id: %s", node.id)
+        data_nodes.pop(node.id)
+        # node added/removed events also happen on (re)starts of hass/mqtt/ozw
+        # cleanup device/entity registry if we know this node is permanently deleted
+        # entities itself are removed by the values logic
+        if node.id in removed_nodes:
+            hass.async_add_job(handle_remove_node(hass, node.id))
```

Comment:
Use `hass.async_create_task` instead of `hass.async_add_job`. The latter is legacy for general use.

Reply:
OK, thanks!